### PR TITLE
Update internet-endpoints.md

### PR DESCRIPTION
**Pull request by BalaDelli**

### CODE REVIEW

**Comment by traya1** [comment]
Learn Build status updates of commit _[8f9ffd9](https://github.com/BalaDelli/memdocs/commits/8f9ffd910fee343554d82f43006ac8e3fc532545)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[memdocs/configmgr/core/plan-design/network/internet-endpoints.md](https://github.com/BalaDelli/memdocs/blob/patch-1/memdocs/configmgr/core/plan-design/network/internet-endpoints.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/aa268e4d-aaaf-30c8-4118-35d62ca794c9/202212161734447133-3464/BuildReport?accessString=2e58e0d5289b402f934f188a75b053496e2a5adc62e49bbf7a978388a0004de6).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/aa268e4d-aaaf-30c8-4118-35d62ca794c9?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by BalaDelli** [comment]
#sign-off

**Comment by Banreet** [comment]
Hey Bala, not sure which PR is this. Do you need any help with it?

Thanks,
Banreet Kaur
Product Manager, Microsoft Intune
Mobile: +91 8283836790
LinkedIn Profile<https://www.linkedin.com/in/banreetkaur/>

[MSFT_logo_Gray DE sized SIG1.png]

**Comment by Banreet** [comment]
#sign-off

File: memdocs/configmgr/core/plan-design/network/internet-endpoints.md [hunk 1]
```diff
@@ -179,6 +179,7 @@ If you use Configuration Manager to deploy and update Microsoft 365 Apps for ent
 - `officecdn.microsoft.com` to synchronize the software update point for Microsoft 365 Apps for enterprise client updates
 
 - `config.office.com` to create custom configurations for Microsoft 365 Apps for enterprise deployments
+-  https://clients.config.office.net & https://go.microsoft.com/fwlink/?linkid=2190568 to support deploying updates for Microsoft 365 Apps for enterprise
```
Comment (JohanFreelancer9) [inline memdocs/configmgr/core/plan-design/network/internet-endpoints.md:182]:
```suggestion

-  `https://clients.config.office.net & https://go.microsoft.com/fwlink/?linkid=2190568` to support deploying updates for Microsoft 365 Apps for enterprise
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @BalaDelli, kindly commit the suggested changes. Many thanks.

**Comment by BalaDelli** [comment]
#Sign-off

**Comment by Banreet** [comment]
#sign-off

**Comment by traya1** [comment]
Learn Build status updates of commit _[8f9ffd9](https://github.com/BalaDelli/memdocs/commits/8f9ffd910fee343554d82f43006ac8e3fc532545)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[memdocs/configmgr/core/plan-design/network/internet-endpoints.md](https://github.com/BalaDelli/memdocs/blob/patch-1/memdocs/configmgr/core/plan-design/network/internet-endpoints.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/aa268e4d-aaaf-30c8-4118-35d62ca794c9/202301031628538970-3464/BuildReport?accessString=ae04f4141b6870a02805f8d8e29119adc76bb4cd8d14681c33859fc9596d6995).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/aa268e4d-aaaf-30c8-4118-35d62ca794c9?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Review by Banreet — APPROVED** [review]

**Comment by Banreet** [comment]
#sign-off

File: memdocs/configmgr/core/plan-design/network/internet-endpoints.md (hunk 1 above)
Comment (JohanFreelancer9) [inline memdocs/configmgr/core/plan-design/network/internet-endpoints.md:182]:
```suggestion

-  `https://clients.config.office.net` and `https://go.microsoft.com/fwlink/?linkid=2190568` to support deploying updates for Microsoft 365 Apps for enterprise
```

**Comment by Banreet** [comment]
#sign-off

**Pull request merged**